packages/deployments/contracts/contracts/messaging/RootManager.sol

```diff
@@ -25,16 +26,24 @@ contract RootManager is ProposedOwnable, IRootManager, WatcherClient, DomainInde
 
   event DelayBlocksUpdated(uint256 previous, uint256 updated);
 
-  event RootAggregated(uint32 domain, bytes32 receivedRoot, uint256 queueIndex);
+  event RootReceived(uint32 domain, bytes32 receivedRoot, uint256 queueIndex);
```

Comment:
we should make an issue flagging these event changes

Reply:
agreed